src/components/Dashboard/AppointmentElement/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import Wrapper from '@components/Wrapper';
+import { Stack, Box } from '@mui/system';
+import { Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { authApi } from 'services/AuthService';
+
+import { ReactComponent as CameraIcon } from '@assets/Camera.svg';
+import { ACTIVE, ANOTHER_FUCKING_BLUE, BORDER } from '@constants/colors';
+import { VERY_SMALL_FONT_SIZE } from '@constants/fontSizes';
+
+interface AppointmentElementProps {
+  index: number;
+  remoteDoctor: Record<string, any>;
+  patient: Record<string, any>;
+}
+
+const AppointmentElement = ({
+  index,
+  remoteDoctor,
+  patient,
+}: AppointmentElementProps) => {
+  const [show, setShow] = React.useState<boolean>(false);
+
+  const { t } = useTranslation();
+
+  const fullText = patient.notes[0].note;
```

Comment:
1) what if you will not have patient.notes[0]

Reply:
fixed